server/src/migrations/20210816112955_addInvitations.ts

```diff
@@ -0,0 +1,18 @@
+import * as Knex from 'knex';
+
+export async function up(knex: Knex): Promise<any> {
+  return knex.schema.createTable('invitations', (table) => {
+    table.uuid('id').primary().index();
+    table
+      .integer('roadmapId')
+      .references('id')
+      .inTable('roadmaps')
+      .onDelete('CASCADE');
+    table.integer('type');
+    table.string('email', 75);
```

Comment:
To be consistent with the json schema and other email fields in the db
```suggestion
    table.string('email', 255);
```